anomalib/utils/loggers/console.py

```diff
@@ -0,0 +1,110 @@
+"""Console Logger."""
```

Comment:
I'm not sure I completely understand why this file is needed. Can't we just specify the format using 
`logging.basicConfig(format="%(asctime)s - %(name)s - %(levelname)s - %(message)s", level=logging.INFO)`
in the entrypoint scripts, and create the individual loggers in each file with `logging.getLogger(__name__)`?